designer/client/src/containers/DynamicTab.tsx

```diff
@@ -50,11 +54,22 @@ export const RemoteModuleTab = <CP extends RemoteComponentProps>({
     );
 };
 
-export const IframeTab = ({ tab }: { tab: Pick<DynamicTabData, "addAccessTokenInQueryParam" | "url"> }) => {
-    const { addAccessTokenInQueryParam, url } = tab;
-    const accessToken = addAccessTokenInQueryParam && SystemUtils.getAccessToken();
+export const IframeTab = ({ tab }: { tab: Pick<DynamicTabData, "addAccessTokenInQueryParam" | "accessTokenInQuery" | "url"> }) => {
```

Comment:
maybe better to write it that way?
`type IframeTab = { tab: Pick<DynamicTabData, "addAccessTokenInQueryParam" | "accessTokenInQuery" | "url"> }`

`export const IframeTab = ({ tab }: IframeTab) => {`